test/fixtures/fake_protoc

```diff
@@ -26,7 +26,7 @@ main() {
     # fix out_dir in case protoc was run for the go plugin
     local out_dir=$(echo "$out_dir" | sed 's/.*--go_out=plugins=grpc:\([^ ]*\).*/\1/')
     # fix out_dir in case protoc was run for the python plugin
-    local out_dir=$(echo "$out_dir" | sed 's/.*--grpc_python_out=\([^ ]*\).*/\1/')
+    local out_dir=$(echo "$out_dir" | sed 's/.*--python_out=\([^ ]*\).*/\1/')
```

Comment:
nit: out_dir is already declared as `local` previously, I think you don't have to declare it again, just update it.